pkg/raft/tracker/progress.go

```diff
@@ -433,3 +433,13 @@ func (m ProgressMap) String() string {
 	}
 	return buf.String()
 }
+
+// BasicProgress contains a subset of fields from Progress.
+type BasicProgress struct {
+	// Match corresponds to Progress.Match.
+	Match uint64
+	// Next corresponds to Progress.Next.
+	Next uint64
+	// State corresponds to Progress.State.
+	State StateType
```

Comment:
nit: `State` should go first (same as in `Progress`) because it defines the behaviour.
Same in places where the field is populated.

Reply:
Mistaken, `State` is not first in `Progress` (though it should be). Feel free to ignore.